src/Microsoft.Diagnostics.Monitoring.HostingStartup/ParameterCapturing/FunctionProbes/ProfilerAbi.cs

```diff
@@ -50,6 +51,9 @@ internal struct ParameterBoxingInstructions
         //
         public uint Token;
 
+        public byte* SignatureBufferPointer;
+        public uint SignatureBufferLength;
```

Comment:
I *could* remove this field and make it re-use our existing uint field above, but I'd prefer to keep these fields separate for now for readability. If folks have strong preferences for keeping this struct's memory size to an absolute minimum let me know and we can look into removing this field.

Reply:
I'd rather not store buffer sizes in unions, let's keep them separate.